frontend/components/customizer/customizer.tsx

```diff
@@ -11,6 +12,57 @@ import {
   CopiedContentIndicatorHandle,
 } from './copied-content-indicator'
 
+
+function generateBuiltBadgeUrl({
+  baseUrl,
+  queryString,
+  path,
+}: {
+  baseUrl: string,
+  queryString: string | undefined,
+  path: string,
+}): string {
+  const suffix = queryString ? `?${queryString}` : ''
+  return `${baseUrl}${path}${suffix}`
+}
+
+function renderLivePreview({
+  baseUrl,
+  queryString,
+  path,
+  pathIsComplete,
+}: {
+  baseUrl: string,
+  queryString: string | undefined,
+  path: string,
+  pathIsComplete: boolean | undefined, 
+}): JSX.Element {
+  let src
+  const debounced = debounce(function() {
+    return generateBuiltBadgeUrl({
+       baseUrl,
+       queryString,
+       path
+     })
+ }, 1000, {leading: true})
```

Comment:
The way React works is that it repeatedly renders, meaning this function is going to be reinvoked frequently. For debounce to work correctly, you need to invoke it only once and then re-invoke the function it returns. But here you're creating a new `debounced` function on every render.

One way to fix this is to use `useCallback` which will store the debounced function instead of creating a new one on each render. (I am pretty sure `useCallback` will work here. If not, then you could use `useRef`, which is the most generic way to persist a value across all renders.)

Alternatively, you could try doing it without lodash.